features/android/support/patches.rb

```diff
@@ -0,0 +1,27 @@
+require "calabash-android"
+
+=begin
+These methods are implemented for Calabash-ios.
+We are implementing their Android equivalents if possible so some of them may be empty.
+=end
+
+
+module Calabash
+  module Android
+    module WaitHelpers
+      def wait_for_none_animating(*args)
+        sleep 1
```

Comment:
this can be done differently. e.g:

```
# waits for absence of loading spinner. Will raise an error after 45 seconds if spinner is still on the screen.
def wait_for_none_animating
  Timeout.timeout(45) do
    sleep 1
    while element_exists("* marked:'sv_loading_content'") || element_exists('CircleImageView') || element_exists('ProgressBar')
      sleep 0.7
    end
  end
rescue Timeout::Error
  raise 'The page failed to load.'
end
```

In this way(by finding progress bars in your app) it will not fail randomly if your API or connection is slow